packages/react-native-editor/android/app/build.gradle

```diff
@@ -107,7 +108,13 @@ def enableProguardInReleaseBuilds = false
  */
 def enableHermes = project.ext.react.get("enableHermes", false);
 
+/**
+ * Architectures to build native code for in debug.
+ */
+def nativeArchitectures = project.getProperties().get("reactNativeDebugArchitectures")
+
 android {
+    ndkVersion rootProject.ext.ndkVersion
```

Comment:
My understanding is that we don't really need to specify the NDK version as
1. it's implied by the Android Gradle Plugin (AGP)
2. [The version we specify](https://github.com/WordPress/gutenberg/blob/553f1fda0724fa473c381c26c2aebb97655e8f1f/packages/react-native-editor/android/build.gradle#L11) does actually match [the one implied by AGP](https://developer.android.com/studio/projects/install-ndk) (`21.4.7075529` for AGP `v4.2.2`).

That said, I don't think it hurts either, and since specifying helps minimize the diff versus the upgrade-helper, I'm +1 with keeping it in place. Let me know if you have thoughts on this David, thanks!

Reply:
Ah, good to know. As you referenced, this addition originates from the [React Native Upgrade Helper](https://react-native-community.github.io/upgrade-helper/). My lack of Android experience limited my ability to identify that it is not entirely necessary. 

Provided the code has no negative impact — in build, runtime, or significant maintenance burden — I do favor retaining the code to simplify the upgrade process of React Native through more straightforward diff comparison.